tests/CMakeLists.txt

```diff
@@ -8,4 +8,4 @@ add_library(testdll3 SHARED testdll3.c)
 set_target_properties(testdll3 PROPERTIES PREFIX "")
 add_executable(t_dlfcn test.c)
 target_link_libraries(t_dlfcn dl)
-add_test(NAME t_dlfcn COMMAND $<TARGET_FILE:t_dlfcn> WORKING_DIRECTORY $<TARGET_FILE_DIR:t_dlfcn>)
+add_test(NAME t_dlfcn COMMAND ${WRAPPER} $<TARGET_FILE:t_dlfcn> WORKING_DIRECTORY $<TARGET_FILE_DIR:t_dlfcn>)
```

Comment:
If we are cross compiling and wine is not used then we should not run test. But we should compile test (compilation is also some kind of test that it can be compiled).